ec2_restart_instance_by_filter_and_az/README.md

```diff
@@ -0,0 +1,96 @@
+# Restart an [EC2][ElasticComputeCloud] instance selected by filter Chaos Experiment
+
+This experiment explores whether my application can maintain itself when an [EC2][ElasticComputeCloud] instance is restarted, when it is selected by a filter.
+
+[ElasticComputeCloud]: https://aws.amazon.com/ec2/
+
+## Setup for Experiment Execution
+
+### Install the Chaos Toolkit
+
+This experiment uses the [free and open source Chaos Toolkit][chaostoolkit]. The instructions for installing the [Chaos Toolkit][chaostoolkit] are available in the [project's documentation][docs].
+
+[chaostoolkit]: https://chaostoolkit.org/
+[docs]: https://docs.chaostoolkit.org
+
+### Install Any Required Drivers
+
+The [Chaos Toolkit][chaostoolkit] is extended by adding [drivers]. This experiment requires the following drivers:
+
+* Chaos Toolkit Extension for AWS [aws-driver] [aws-driver-github]
+
+[drivers]: https://docs.chaostoolkit.org/drivers/overview/
+[aws-driver]: https://docs.chaostoolkit.org/drivers/aws/
+[aws-driver-github]: https://github.com/chaostoolkit-incubator/chaostoolkit-aws
+
+## Running the Experiment
+
+### Required Parameters
+
+This experiment requires the following parameters:
+
+* Environment Variables
+  * `APPLICATION_ENTRYPOINT_URL` - Specifies the application entry point URL where your application can be reached in your environment.
+  * `INSTANCE_ID` - The id of the instance that will be stopped using a filter.
+  * `AVAILABILITY_REGION` - The availability region of the instance that will be stopped.
```

Comment:
Should this ideally be "AVAILABILITY_ZONE" as that is what it usually means in AWS terminology?